### PR TITLE
Update `iframe` height calculation

### LINKED ISSUE #539: Send all opengraph tags to the parent iframe
To improve SEO in web crawlers, like google, we should send all meta tags like title, description, and Open Graph tags, to the parent wordpress.org/openverse site in a postMessage event. 

This should _theoretically_ give Google's crawler the information, as it executes client-side JavaScript, but this would _also_ enable _another idea_ I had to fix the SEO problems!

The Openverse WordPress theme could take this postmessage seo information, cache/store it in the database, _and then_ render it server-side on subsequent page views! This would fix SEO problems for users in things like link previews in Slack and other places opengraph tags are used.

Maybe next, we'll postmessage the entire html response from the Openverse iframe and cache it in PHP 😆

### PR DESCRIPTION
**Pull request by dhruvkb**
This PR
- changes supplied path to lowercase to prevent 404 on case variations of `https://wordpress.org/openverse`
- uses a fixed value of `--header-height` to make the `iframe` occupy 100vh - header height
- adds functionality for sending meta-data from `iframe` to the parent window
- updates test page (also adds tips for certain fields)
- updates `README.md` and `.wp-env.json` to setup a dev environment

Closes https://meta.trac.wordpress.org/ticket/6041

Partially fixes WordPress/openverse-frontend#539

### CODE REVIEW

**Comment by dhruvkb** [comment]
^ Accidentally clicked the 'Close' button, sorry.

**Comment by sarayourfriend** [comment]
Maybe I'm doing something wrong but I can't get the theme to run locally.

> Fatal error: Uncaught Error: Call to undefined function WordPressdotorg\skip_to() in /var/www/html/wp-content/themes/wporg-openverse/header.php:14 Stack trace: #0 /var/www/html/wp-includes/template.php(770): require_once() #1 /var/www/html/wp-includes/template.php(716): load_template('/var/www/html/w...', true, Array) #2 /var/www/html/wp-includes/general-template.php(48): locate_template(Array, true, true, Array) #3 /var/www/html/wp-content/themes/wporg-openverse/index.php(17): get_header() #4 /var/www/html/wp-includes/template-loader.php(106): include('/var/www/html/w...') #5 /var/www/html/wp-blog-header.php(19): require_once('/var/www/html/w...') #6 /var/www/html/index.php(17): require('/var/www/html/w...') #7 {main} thrown in /var/www/html/wp-content/themes/wporg-openverse/header.php on line 14 

I can't find `skip_to` defined in this repository, I'm not sure where it's meant to come from.

Anyone else running into this?

**Comment by dhruvkb** [comment]
@sarayourfriend I'm updating the instructions in the `README.md` based on the help from @iandunn and @ryelle. Should be ready in a short bit.

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css [hunk 1]
```diff
@@ -1,8 +1,12 @@
+:root {
+    --header-height: 104.453px; /* redesiged global-header */
```
Comment (zackkrida) [inline wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css]: Oh dear, it looks like the header is actually _taller_ 117px, on mobile. The breakpoint is `890px`.

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/js/iframe_nav.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 document.addEventListener('DOMContentLoaded', () => {
   const { pathname: path, search: query } = window.location; // starts with /
 
-  let iframePath = path.replace(openverseSubpath, '').replace(/^\/$/, ''); // Remove Openverse site subpath
+  let iframePath = path
+    .toLocaleLowerCase()
```
Comment (zackkrida) [inline wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/js/iframe_nav.js:5]: great to get this in!

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css (hunk 1 above)
Comment (dhruvkb) [reply wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css]: Good note, are there any other breakpoints with different height?

Comment (ryelle) [reply wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css]: 890 is the only breakpoint that changes the header height

**Comment by dhruvkb** [comment]
@sarayourfriend I've updated the dev instructions.

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css (hunk 1 above)
Comment (zackkrida) [reply wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css]: thanks!

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md [hunk 3]
```diff
@@ -26,7 +26,27 @@ Follow these steps to set up a local playground for the theme:
     4.  `cd` back to the Openverse theme directory at `../wporg-openverse`
         (i.e. `wordpress.org/wordpress.org/public_html/wp-content/themes/pub/wporg-openverse`).
 
-2.  You can choose to set up a new environment automatically or work in an
+2.  Build the MU plugins.
+
+    1.  Clone the `WordPress/wporg-mu-plugins` repo right into this directory.
+        It's `.gitignored` so it shouldn't affect anything.
```
Comment (sarayourfriend) [inline wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md:32]: Can you clarify exactly which directory to do this in?

Also, I can't find any gitignore files in this repository at all...

Comment (dhruvkb) [reply wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md:32]: This is the `.gitignore` I'm referring to `wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/.gitignore`.

You should clone inside `wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/`.

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css (hunk 1 above)
Comment (dhruvkb) [reply wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css]: Updated in 4ce3c2e.

File: wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css [hunk 4]
```diff
@@ -1,8 +1,19 @@
+:root {
+    --header-height: 117px;
+}
+
+@media (min-width: 890px) {
+    :root {
+        --header-height: 104.453;
```
Comment (zackkrida) [inline wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css:7]:
```suggestion
        --header-height: 104.453px;
```

**Comment by iandunn** [comment]
Merged in https://meta.trac.wordpress.org/changeset/11466, with Zack's `px` suggestion included

**Comment by dhruvkb** [comment]
> with Zack's `px` suggestion included

Thanks @iandunn!

**Pull request closed without merging**